README.md

```diff
@@ -434,7 +515,7 @@ The great [`click-odoo`][] scripting framework and the collection of scripts fou
 
 \* Note: This replaces the deprecated `python-odoo-shell` binary.
 
-### [`nano`][]
+### [`nano`](https://www.nano-editor.org/)
```

Comment:
Shouldn't links from headers be removed to avoid problems?

Reply:
It works in this case, i.e. when the link is entire header